addlicense/main.go

```
@@ -488,7 +488,27 @@ func hasLicense(b []byte) bool {
 	if len(b) < 1000 {
 		n = len(b)
 	}
-	return bytes.Contains(bytes.ToLower(b[:n]), []byte("copyright")) ||
-		bytes.Contains(bytes.ToLower(b[:n]), []byte("mozilla public")) ||
-		bytes.Contains(bytes.ToLower(b[:n]), []byte("spdx-license-identifier"))
+
+	// Default conditions
+	defaultTerms := []string{"hashicorp, inc.", "mozilla public", "spdx-license-identifier"}
+
+	// Load conditions from config file
+	configPath := "../LicenceCopywrite.json"
+	file, err := os.ReadFile(configPath)
+	if err == nil {
```

Suggestion: We should add the capability for this function to return errors

Instead of having the function silently failing and using default terms, we should let the user know that the `Readfile` function failed